### PR TITLE
CONN-6904 Update quick start for Snowflake Connector for ServiceNow to match v2 flow

### PR DESCRIPTION
**Pull request by sfc-gh-hlegec**
As we're close to GA of Snowflake Connector for ServiceNow we want to refresh the quick start guide to match the refreshed flow.
This change should be released just after the GA is announced.

### CODE REVIEW

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md [hunk 1]
```diff
@@ -49,29 +49,30 @@ Deployment is usually pretty quick, around five minutes. But, while you wait, le
...
+1. Log on to your Snowflake account through the Snowsight web interface and change to the **ORGADMIN** role. 
 1. Select “Admin -> Billing & Terms”.
 4. In the “Snowflake Marketplace” section, review the Consumer Terms of Service.
 5. If you agree to the terms, select “Accept Terms & Conditions”.
 
 ### Set up a Virtual Warehouse
 Duration: 1
 
-You'll need some compute for the connector, so let's set up a virtual warehouse to do that. A second virtual warehouse will be created automatically in the configure section.
+You'll need some compute for the connector installation process, so let's set up a virtual warehouse to do that. A second virtual warehouse will be created automatically in the configure section.
```
Comment (sfc-gh-pzalas) [inline site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: "A second virtual warehouse will be created automatically in the configure section." - I wonder if this note is needed here. If I'm fresher to the connector, I don't know anything about "configure section". Why I need to care about this second warehouse?

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md [hunk 2]
```diff
@@ -81,92 +82,107 @@ The connector, the first of its kind to be deployed on Snowflake's native apps f
...
+You can also provide names of existing objects.
+
+![default config](assets/configure_defaults.png)
+
+Check out [Configuring the Snowflake Connector for ServiceNow®](https://other-docs.snowflake.com/en/connectors/servicenow/v2/installing-snowsight-2.0#configuring-the-sncv2) for more information on these fields.
+
+Select **Configure**. It can take a few minutes for the configuration process to complete, and you will be moved to the next step.
+
+> aside negative
+> Watch out!!! The created warehouse is created as a **Large** and with a auto suspension after 10 minutes. So this means, if you set to refresh every hour, the Large warehouse (8 credits/hour) will wake up for a minimum of 10 minutes every hour.  For this lab, you don't need all the power! Go to Admin-> Warehouses -> SERVICENOW_WAREHOUSE -> ... > Edit, and change this to an XSMALL, and the auto timeout to one minute. In a real-life use case, a Large warehouse size is often needed.
```
Comment (sfc-gh-pzalas) [inline site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: Missing space: `Admin-> Warehouses`. Similar comment few lines below

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md [hunk 3]
```diff
@@ -49,29 +49,30 @@ Deployment is usually pretty quick, around five minutes. But, while you wait, le
...
 Duration: 1
 
-You'll need some compute for the connector, so let's set up a virtual warehouse to do that. A second virtual warehouse will be created automatically in the configure section.
+You'll need some compute for the connector installation process, so let's set up a virtual warehouse to do that. A second virtual warehouse will be created automatically in the configure section.
 
-Change to the **accountadmin** role.
+Change to the **ACCOUNTADMIN** role.
 1. Navigate to Admin -> Warehouses and select **+ Warehouse**. 
-2. Name the  virtual warehouse **SERVICENOW_CONNECTOR_WH**, size XS, and, leaving the defaults, select **Create Warehouse**. 
+2. Name the virtual warehouse **SERVICENOW_CONNECTOR_WH**, size XS, and, leaving the defaults, select **Create Warehouse**.
```
Comment (sfc-gh-pzalas) [inline site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: Later we create `SERVICENOW_WAREHOUSE`. Maybe let's pick a different warehouse name than `SERVICENOW_CONNECTOR_WH`?

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md [hunk 4]
```diff
@@ -81,92 +82,107 @@ The connector, the first of its kind to be deployed on Snowflake's native apps f
...
+Select **Configure**. It can take a few minutes for the configuration process to complete, and you will be moved to the next step.
+
+> aside negative
+> Watch out!!! The created warehouse is created as a **Large** and with a auto suspension after 10 minutes. So this means, if you set to refresh every hour, the Large warehouse (8 credits/hour) will wake up for a minimum of 10 minutes every hour.  For this lab, you don't need all the power! Go to Admin-> Warehouses -> SERVICENOW_WAREHOUSE -> ... > Edit, and change this to an XSMALL, and the auto timeout to one minute. In a real-life use case, a Large warehouse size is often needed.
+
+> aside positive
+> Absolutely attach a resource monitor to the SERVICENOW_WAREHOUSE. Go to Admin->Resource Monitors->+ Resource Monitor, and create a warehouse resource monitor:
+![resource monitor](assets/monitor.png)
+
+## Set up the Snowflake to ServiceNow® Oauth hand-shake
```
Comment (sfc-gh-pzalas) [inline site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: `Oauth` -> `OAuth2`

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md [hunk 5]
```diff
@@ -81,92 +82,107 @@ The connector, the first of its kind to be deployed on Snowflake's native apps f
...
 Duration: 1
 
-1. Paste the  **Client id** from ServiceNow® into the Snowflake configure pop-up.
-1. Go back to the ServiceNow® tab and copy the **Client secret** and paste it into the Snowflake configure pop-up. 
-1. No need to change the Advanced Settings, but feel free to check them out.
+1. Select **OAuth2** as an authentication method
+1. Fill in the ServiceNow® instance details. This is the first part of the ServiceNow® URL for your ServiceNow® account, **without** the trailing *service-now.com*.
```
Comment (sfc-gh-pzalas) [inline site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: and without `https://`

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md [hunk 6]
```diff
@@ -178,22 +194,19 @@ Duration: 1
 > - Incremental updates occur only for tables with **sys_updated_on** or **sys_created_on** columns.
 > - For tables that do not have sys_updated_on or sys_created_on columns, the connector uses **truncate and load** mode. In this mode, the table is always ingested using the initial load approach, and newly ingested data replaces the old data.
 
-1. In the **Snowflake Connector for ServiceNow** window, under the status for the connector, which displays "Start Data Sync", select **Select Tables**.
+1. In the **Snowflake Connector for ServiceNow** window, on the top bar, select **Data Sync**.
 
 1. To be able to run our test query later, we need to ingest a couple of tables. From the search window enter **incident** and check the box next to it and choose a 30 minute sync time.
```
Comment (sfc-gh-pzalas) [inline site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md:201]: I think you need to click "Edit tables" to select tables for ingestion.

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md [hunk 7]
```diff
@@ -219,43 +232,33 @@ SELECT * FROM SNOWFLAKE_CONNECTOR_FOR_SERVICENOW.public.connector_stats;
...
-CREATE ROLE IF NOT EXISTS servicenow_reader_role;
-GRANT USAGE ON DATABASE SERVICENOW_DEST_DB TO ROLE servicenow_reader_role;
-GRANT USAGE ON SCHEMA DEST_SCHEMA TO ROLE servicenow_reader_role; 
-GRANT SELECT ON FUTURE TABLES IN SCHEMA DEST_SCHEMA TO ROLE servicenow_reader_role;
-GRANT SELECT ON FUTURE VIEWS IN SCHEMA DEST_SCHEMA TO ROLE servicenow_reader_role;
-GRANT SELECT ON ALL TABLES IN SCHEMA DEST_SCHEMA TO ROLE servicenow_reader_role;
-GRANT SELECT ON ALL VIEWS IN SCHEMA DEST_SCHEMA TO ROLE servicenow_reader_role;
-GRANT USAGE ON WAREHOUSE SERVICENOW_WAREHOUSE TO ROLE servicenow_reader_role;
-```
+The connector exposes an application role named `DATA_READER`.
```
Comment (sfc-gh-pzalas) [inline site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: There is information that the role exists, but what it does?

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md [hunk 8]
```diff
@@ -338,20 +336,21 @@ During this lab, we're only ingesting the data, so it makes sense to stop the in
...
 
-Read the warning and select **Stop Ingestion**.
+If you completed the experiment or for any reason no longer need the connector you can easily uninstall it via the Snowflake Marketplace.
 
-## Uninstall the Connector (but not the data)
-If you are using the public preview connector, make sure to check out the limitations, one of which is during the preview period, before the connector is generally available, Snowflake will release an update that requires you to **uninstall** and reinstall the connector from the Snowflake Marketplace.
+1. Select **Data Products** and then **Apps**.
+1. Select three dots icon in the item on the list representing the connector app.
+1. Select **Uninstall**
+1. Decide if you want to delete the objects owned by the application (tables and views in the destination schema) or transfer ownership of them to another role
```
Comment (sfc-gh-pzalas) [inline site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: Nit: maybe it's worth to stress more that objects owned by the application include ingested data, e.g.: Decide if you want to delete the objects owned by the application (including tables and views with ingested data in the destination schema) or transfer ownership of them to another role

Comment (sfc-gh-hlegec) [reply site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: added

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md (hunk 7 above)
Comment (sfc-gh-hlegec) [reply site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: Information that it has read access to the ingested data added

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md (hunk 5 above)
Comment (sfc-gh-hlegec) [reply site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: done

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md (hunk 4 above)
Comment (sfc-gh-hlegec) [reply site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: fixed

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md (hunk 3 above)
Comment (sfc-gh-hlegec) [reply site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: renamed to `CONNECTOR_UI_WH`

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md (hunk 2 above)
Comment (sfc-gh-hlegec) [reply site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: fixed

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md (hunk 1 above)
Comment (sfc-gh-hlegec) [reply site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: Comment moved to the place where second warehouse is created

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md (hunk 6 above)
Comment (sfc-gh-hlegec) [reply site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md:201]: On the `Data sync` tab, when you don't have any tables selected, edit view is opened by default.

Comment (sfc-gh-pzalas) [reply site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md:201]: Ok

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md (hunk 3 above)
Comment (sfc-gh-pzalas) [reply site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: LGTM

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md (hunk 2 above)
Comment (sfc-gh-pzalas) [reply site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: LGTM

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md (hunk 5 above)
Comment (sfc-gh-pzalas) [reply site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: add "and" in `**without** *https://* protocol the trailing *service-now.com*.` part

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md (hunk 4 above)
Comment (sfc-gh-pzalas) [reply site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: Ok, but my eyes are still bleeding on second letter small in `Oauth`

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md (hunk 8 above)
Comment (sfc-gh-pzalas) [reply site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: Ok

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md (hunk 7 above)
Comment (sfc-gh-pzalas) [reply site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: Ok

**Review by sfc-gh-pzalas — APPROVED** [review]

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md (hunk 5 above)
Comment (sfc-gh-hlegec) [reply site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: done

File: site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md (hunk 4 above)
Comment (sfc-gh-hlegec) [reply site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md]: fixed

**Pull request merged**